src/lib/components/Feedback.svelte

```diff
@@ -13,6 +14,10 @@
     async function handleSubmit() {
         submitting = true;
         error = undefined;
+
+        const cloudUserId = loggedIn && $user?.$id ? $user.$id : undefined;
+        const cloudUserEmail = loggedIn && $user?.email ? $user.email : undefined;
```

Comment:
Why not auto-populate the email field?

Reply:
Do we need userId in that case then?

Reply:
I've kept the ID for now, lmk if thats fine else we can remove.